model-optimizer/mo/offline_transformations.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+"""
+ Copyright (C) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import sys
+
+
+if __name__ == "__main__":
+    try:
+        from openvino.inference_engine import IECore
+        from openvino.offline_transformations import ApplyMOCTransformations
+    except ImportError as e:
+        print("[ WARNING ] offline_transformations import error: {}".format(e))
+        exit(1)
+
+    orig_model_name = sys.argv[1]
+    ie = IECore()
+    net = ie.read_network(model=orig_model_name + ".xml", weights=orig_model_name + ".bin")
```

Comment:
We removed serialization, but still require reading functionality. Do we have clear statistics that confirms no one model suffers from too expensive conversion time? I believe all should be fine in comparison to overall conversion time.

Consider another risk: if a network has custom operations that cannot be read w/o providing dedicated extensions to IECore registered with https://docs.openvinotoolkit.org/latest/classInferenceEngine_1_1Core.html#aac8284a60791abd6e50ddab0c695e38f. read_network will report error in this case, how is it going to be reported to the user?

Reply:
We had some troubles with memory consumption (they were actually fixed) but to minimize risk I suggest not to use read_network in 21.3 even for testing purposes.